public/app/features/dashboard/components/Inspector/InspectStatsTable.tsx

```diff
@@ -1,20 +1,32 @@
 import React from 'react';
-import { FieldType, formattedValueToString, getDisplayProcessor, QueryResultMetaStat, TimeZone } from '@grafana/data';
+import {
+  FieldType,
+  formattedValueToString,
+  getDisplayProcessor,
+  GrafanaTheme,
+  QueryResultMetaStat,
+  TimeZone,
+} from '@grafana/data';
 import { DashboardModel } from 'app/features/dashboard/state';
 import { config } from 'app/core/config';
+import { stylesFactory, useTheme } from '@grafana/ui';
+import { css } from 'emotion';
 
 interface InspectStatsTableProps {
   dashboard: DashboardModel;
   name: string;
   stats: QueryResultMetaStat[];
 }
 export const InspectStatsTable: React.FC<InspectStatsTableProps> = ({ dashboard, name, stats }) => {
+  const theme = useTheme();
+  const styles = getStyles(theme);
+
   if (!stats || !stats.length) {
     return null;
   }
 
   return (
-    <div style={{ paddingBottom: '16px' }}>
+    <div className={styles.wrapper}>
```

Comment:
Great work! nit: You missed the `<td style={{ textAlign: 'right' }}>`

Reply:
lol